dev/cnf/oss_dependencies.maven

```diff
@@ -177,7 +177,7 @@ org.apache.felix:org.apache.felix.gogo.runtime:0.16.2
 org.apache.felix:org.apache.felix.gogo.shell:0.12.0
 org.apache.felix:org.apache.felix.http.api:2.2.0
 org.apache.felix:org.apache.felix.http.base:2.2.0
-org.apache.felix:org.apache.felix.scr:2.1.16
+org.apache.felix:org.apache.felix.scr:2.1.17.ibm20190708-1
```

Comment:
Because this is a build of a snapshot of org.apache.felix.scr we must put it in a group id starting with com.ibm.ws (e.g. com.ibm.ws.org.apache.felix)

Reply:
done